### PR TITLE
first pass at WKO name change

### PR DESCRIPTION
**Pull request by rosemarymarano**
Lots of instances, headings, scripts/samples, cross-references to change. Hoping that I got them all.
Oracle WebLogic Server Kubernetes Operator -> WebLogic Kubernetes Operator

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ submit your pull request.
 If you have any questions about a possible submission, feel free to open
 an issue too.
 
-## Contributing to the Oracle WebLogic Server Kubernetes Operator repository
+## Contributing to the WebLogic Kubernetes  Operator repository
```
Comment (rjeberhard) [inline CONTRIBUTING.md:12]: Extra space added

Comment (rjeberhard) [reply CONTRIBUTING.md:12]: I think you left this in to see if we were really paying attention.

File: kubernetes/hands-on-lab/tutorials/deploy.weblogic_short.ocishell.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Oracle WebLogic Server Kubernetes Operator Tutorial #
+# WebLogic Kubernetes  Operator Tutorial #
```
Comment (rjeberhard) [inline kubernetes/hands-on-lab/tutorials/deploy.weblogic_short.ocishell.md:1]: Another extra space :)

File: kubernetes/hands-on-lab/tutorials/install.operator.ocishell.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
-# Oracle WebLogic Server Kubernetes Operator Tutorial #
+# WebLogic Kubernetes  Operator Tutorial #
```
Comment (rjeberhard) [inline kubernetes/hands-on-lab/tutorials/install.operator.ocishell.md:1]: Another... They are starting to be like Easter Eggs.

File: kubernetes/hands-on-lab/tutorials/install.operator.ocishell.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
 
 ### Install and configure the operator  ###
 
-An operator is an application-specific controller that extends Kubernetes to create, configure, and manage instances of complex applications. The Oracle WebLogic Server Kubernetes Operator (the "operator") simplifies the management and operation of WebLogic domains and deployments.
+An operator is an application-specific controller that extends Kubernetes to create, configure, and manage instances of complex applications. The WebLogic Kubernetes  Operator (the "operator") simplifies the management and operation of WebLogic domains and deployments.
```
Comment (rjeberhard) [inline kubernetes/hands-on-lab/tutorials/install.operator.ocishell.md:5]: Next

**Review by rjeberhard — APPROVED** [review]

File: kubernetes/hands-on-lab/tutorials/deploy.weblogic_short.ocishell.md (hunk 2 above)
Comment (rosemarymarano) [reply kubernetes/hands-on-lab/tutorials/deploy.weblogic_short.ocishell.md:1]: You found The TWO places (in the Tutorial and the Contributing doc) that had them. Thank you. Now you are the Eagle Eyes.

File: kubernetes/hands-on-lab/tutorials/install.operator.ocishell.md (hunk 4 above)
Comment (rosemarymarano) [reply kubernetes/hands-on-lab/tutorials/install.operator.ocishell.md:5]: Fixed them all. Thank you.

**Review by mriccell — APPROVED** [review]
Thank you for doing this!

**Pull request merged**